scripts/van_allen_tools/migrate_van_allen_workspaces.py

```diff
@@ -347,6 +347,24 @@ def copy_workspace_entities(destination_workspace_namespace, destination_workspa
     return True, data_table_name_list
 
 
+def get_workspace_attributes(source_workspace_namespace, source_workspace_name):
+    """Copy attributes from source workspace to a destination workspace."""
```

Comment:
This function only gets the workspace attributes right? It doesn't actually do any of the copying work? The description is a bit confusing since I'm expecting to see code for copying it to the destination in here.